### PR TITLE
added new models and migrations for msgboard

### PR DESCRIPTION
**Pull request by simzacks**
Added the new models and migrations to go along with the database lecture.

### CODE REVIEW

**Comment by simzacks** [comment]
@ifireball - I fixed the flake8 problems, except for the ones in 0001_initial.py which is auto generated by the system and has lines too long.
What is the accepted practice for this situation?

**Comment by ifireball** [comment]
does extending the flake8 line-length tolerance to 100-120 not resolve the situation?

**Comment by simzacks** [comment]
It's already at 120

**Comment by simzacks** [comment]
I can add 
`# flake8: noqa`
to the top of the file, so that it's ignored.
This is a question that's going to come up for all of our students, which is why I'd like to have a standard answer for them.

**Comment by ifireball** [comment]
`# flake8: noqa` will switch off all Flak8 checks for the entire file, maybe we can just switch off the line length test

**Comment by simzacks** [comment]
Being that the file is autogenerated do we want any flake8 tests on it?

**Comment by ifireball** [comment]
If someone will inadvertently change it (While resolving a Git conflict for example), we want to catch the situation where it becomes invalid

File: msgboard/migrations/.flake8 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[flake8]
```
Comment (ifireball) [inline msgboard/migrations/.flake8:1]: can we put this in a different PR? also, we probably need to add a section about this in the document

Comment (simzacks) [reply msgboard/migrations/.flake8:1]: It didn't work, so we need to find another solution, in any case. It is directly related to this PR, but I'm not opposed to adding the fix somewhere else.

Comment (simzacks) [reply msgboard/migrations/.flake8:1]: I didn't work because it's expecting it in the root folder. I'll send another PR and then rebase and repush this and see if it works

Comment (simzacks) [reply msgboard/migrations/.flake8]: I wanted to test it in Github, being that it didn't work the first time I added it, so I kept it in this PR.

**Comment by RawanAbuleil** [comment]
ok

**Review by RawanAbuleil — APPROVED** [review]

**Review by RawanAbuleil — APPROVED** [review]

**Comment by ifireball** [comment]
@RawanAbuleil what do you mean to do my marking approval on this PR?

**Comment by ifireball** [comment]
This PR is very outdated by now
